main.py

```diff
@@ -76,6 +77,10 @@
 BLANK_INPUT = CMutableTxIn
 
 
+def no_output(*args, **kwargs):
+    print(*args, file=sys.stderr, **kwargs)
```

Comment:
We need to revisit this since it definitely isn't well named as "no_output."

Reply:
That was the name it had, I was very confused by it :stuck_out_tongue: